### ISSUE TITLE
Create Training, Validation, and Test Sets

### ISSUE THREAD

**Issue by dianzrong** [uoft-tmi/signature-forgery-detection#1]
### Purpose
1. We need to convert our dataset from files into a form we can manipulate using Python
2. We need to divide our dataset into 3:
     a. **Training set** to train our model on (70%)
      b. **Validation set** to see what parameters improve our model (15%)
      c.**Test set** to evaluate how well or model is doing (15%)

### Task
1. Combine the 2 subfolders from [here](https://www.kaggle.com/datasets/shreelakshmigp/cedardataset) into 1 dataset with (binary) labels for whether the signature is forged or not (like in the corresponding resource)
2. Split the combined dataset into 3: one for training (70%), one for validating (15%) and one for testing (15%)

### Resources
1. [Create a Dataset using Tensorflow](https://www.tensorflow.org/tutorials/load_data/images#create_a_dataset)
2. ^ and [Stack Overflow](https://stackoverflow.com/questions/66036271/splitting-a-tensorflow-dataset-into-training-test-and-validation-sets-from-ker) may be useful
